README.md

```diff
@@ -139,6 +140,17 @@ semver
 serve-static
 sinon
 uglify-es
+conventional-changelog-cli
+conventional-changelog-videojs
+doctoc
+husky
+lint-staged
+not-prerelease
+npm-merge-driver-install
+npm-run-all
+shx
+pkg-ok
+in-publish
 ```
 
 All these dependencies should be removable because, in general, build tools are now using the new `videojs-generate-*` libraries. This should make it easier to manage dependencies by offloading much of that work to the Video.js organization. The behavior of these new tools is customizable by editing the files in `scripts/`.
```

Comment:
can we list out the specific videojs-generate packages that replace the above here?

Reply:
sure